src/components/elements/SignUp.jsx

```diff
@@ -124,10 +124,11 @@ function SignUp() {
 
   async function handleSubmit(formData) {
     try {
+      
       await auth.register(formData)
       navigate('/confirmation', { replace: true });
     } catch (e) {
-      window.alert('Something went wrong')
+      window.alert(e)
```

Comment:
We don't want to spit out a verbose image at the user in the case of an error, let's revert this to the Something Went Wrong message and continue from there.

Reply:
This was one of the changes I didn't mean to push up to the repo! Thought I caught all of it, thanks for the second look.